web-test-runner.config.mjs

```diff
@@ -34,6 +34,7 @@ export default {
       '**/special/tacocat/**',
     ],
   },
+  testFramework: { config: { retries: 2 } },
```

Comment:
Is `2` an arbitrary value? Should we start with `1` retry? Large test files take a longer time to run, increasing failing tests 3 times might lead to a big overhead.

Reply:
Yes, it's an arbitrary value - we can also go with `1`